### PR TITLE
new commit new branch

### PR DESCRIPTION
**Pull request by betomoedano**
The app.tsx file didn't have the react hook useState imported!

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import logo from "./logo.svg";
 import "./App.css";
+import React, {useState} from "react";
```
Comment (cesswhite) [inline src/App.tsx]: This is no longer necessary, since we have auto-imports installed
more info: https://github.com/antfu/unplugin-auto-import

**Review by cesswhite — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (betomoedano) [reply src/App.tsx]: Screen Shot 2021-12-10 at 1 04 29 PM

if I remove that line the app still works but I get this ugly red line

Comment (cesswhite) [reply src/App.tsx]: Currently, [unplugin-auto-import](https://github.com/antfu/unplugin-auto-import) has support for all these hooks: [Hooks presets](https://github.com/antfu/unplugin-auto-import/blob/main/src/presets/react.ts)

Comment (cesswhite) [reply src/App.tsx]: Maybe it is a problem with VSCode, it does not affect our development, we will look for a visual solution for that

**Pull request merged**